arch/none/host/src/arch_main.c

```diff
@@ -38,9 +38,7 @@ int scmi_arch_init(void)
 
 int scmi_arch_deinit(void)
 {
-//  TBD: no way to close cleanly the scp firmware
-
-    return 0;
+    return fwk_arch_deinit();
```

Comment:
again,  please create arch/none/host-scmi-server and implement/modify code there